### PR TITLE
i added some misery to alex's file

### PR DESCRIPTION
**Pull request by Shinykittens786**
i added a line of mathematics related misery for children

### CODE REVIEW

File: alexs-file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+abacusabacusabacusabacusabacusabacusabacus
```
Comment (tahmid0517) [inline alexs-file.txt]: Add space between each "abacus".

Comment (tahmid0517) [inline alexs-file.txt]: Add more misery.

Comment (tahmid0517) [inline alexs-file.txt]: Add some math.

**Review by tahmid0517 — CHANGES_REQUESTED** [review]

File: alexs-file.txt (hunk 1 above)
Comment (Shinykittens786) [reply alexs-file.txt]: aw man

Comment (Shinykittens786) [reply alexs-file.txt]: yes sir

Comment (Shinykittens786) [reply alexs-file.txt]: ok 

2+2 = 5

**Review by tahmid0517 — APPROVED** [review]

**Review by tahmid0517 — APPROVED** [review]

**Pull request merged**